docs/release/release-process.md

```diff
@@ -178,26 +250,52 @@ and collect feedback.  Do **not** hit "Publish release" yet.
 You can see and follow the template and structure from [previous
 releases](https://github.com/crossplane/crossplane/releases).
 
-### Stack Release Process
+### Provider Release Process
+
+This section will walk through how to release the Providers and does not
+directly apply to core Crossplane.
+
+#### Pin Provider Dependencies
 
-At this point, we should have a release branch created in all repos with all
-issues resolved that were triaged to be release blockers.  This section will
-walk through how to release the Stacks themselves and does not directly apply to
-core Crossplane.
+Similar to core crossplane, each provider should have its crossplane related
+dependencies pinned to the versions that we are releasing. In the **master**
+branch of each provider repo, update the crossplane and crossplane-runtime
+dependencies to the versions we are releasing.
```

Comment:
Just a note here, we ran into issues in the `v0.9` release process because of an outdated version of `crossplane-tools`. We do not currently version `crossplane-tools` so it is difficult to determine what version you should be pinned to, but in-practice it is usually the latest commit on `master`. Not sure if we want to call that out here, but wanted to make sure it was mentioned :+1:

Reply:
I've added a note about `crossplane-tools` as well, and how in practice we use latest from master.